File: backend/src/main/java/com/wedecom/digito/TestController.java
package com.wedecom.digito;

import org.springframework.web.bind.annotation.*;

@RestController
@RequestMapping("/api")
public class TestController {

    @PostMapping("/{id}/test")
    public String test(@PathVariable String id, @RequestBody String body) {
        System.out.println(id);
        return null;
    }
}

File: backend/src/main/kotlin/com/wedecom/digito/TestController.kt
package com.wedecom.digito

import org.springframework.web.bind.annotation.PathVariable
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class TestController2 {


    @PostMapping("/api2/{id}/test2")
    fun test(@PathVariable id: String) {
        println(id)
    }

}